### ISSUE TITLE
Tamm-Horsfall (uromodulin) casts

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5035]
Definition:A type of acellular urinary cast that are composed only of Tamm-Horsfall glycoprotein, a fact which explains their low refractive index. Hyaline casts may display a spectrum of morphologies, which includes fluffy, compact, convoluted or wrinkled casts. Hyaline casts have a smooth texture and usually have parallel sides with clear margins and blunted ends.
Modified definition: A type of acellular urinary cast that are composed only of Tamm-Horsfall glycoprotein, a fact which explains their low refractive index. Hyaline casts may display a spectrum of morphologies, which includes fluffy, compact, convoluted or wrinkled casts. Hyaline casts have a smooth texture and usually have parallel sides with clear margins and blunted ends.

**Comment by pnrobinson** [comment]
@avizrosenberg
@yongqunh  
@beckysteck

can you please check if Hyaline casts is a synoynm of this new request. If so, should we rename the term and use the above definition?
see:  https://hpo.jax.org/app/browse/term/HP:0031200

**Comment by avizrosenberg** [comment]
Hyaline casts and Tamm Horsfall Casts are synonymous

**Comment by avizrosenberg** [comment]
[PIIS008525381532144X.pdf](https://github.com/obophenotype/human-phenotype-ontology/files/3996169/PIIS008525381532144X.pdf)

**Comment by pnrobinson** [comment]
@avizrosenberg  I added this term with the primary label Intratubular hyaline casts
